src/Domain/Campaign.php

```diff
@@ -77,6 +77,26 @@ public function __toString(): string
         return "Campaign ID #{$this->id}, SFId: {$this->salesforceId}";
     }
 
+    /**
+     * @return bool does this campaign accept ad-hoc giving, i.e. non-regular giving.
+     * @see self::isRegularGiving()
+     */
+    public function isAdHocGiving(): bool
```

Comment:
not sure what we want to call the opposite of regular giving. I've gone with ad-hoc giving but there might be a better term.

Reply:
changed to one-off